src/Microsoft.DotNet.XHarness.Common/Execution/LinuxProcessManager.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.XHarness.Common.Execution
+{
+    public class LinuxProcessManager : UnixProcessManager
+    {
+        [DllImport("/lib/x86_64-linux-gnu/libc.so.6")]
```

Comment:
Remind me why this doesn't use the managed C# process APIs?

Reply:
Please read the PR description

Reply:
please use `[DllImport("libc.so")]` instead since that is way more portable.